### DISCUSSION TITLE
Splash screen still shows Avalonia logo

### DISCUSSION THREAD

**Discussion by AArnott** [AvaloniaUI/Avalonia#16170] [Q&A] [answered]
I replaced the `Resources\drawing\icon.png` file with one that shows my own app icon, where it used to show the Avalonia logo. This got the app icon in the app launcher to show my logo. But the splash screen still shows the Avalonia logo. 
The splash_screen.xml file contains:

```xml
<?xml version="1.0" encoding="utf-8"?>
<layer-list xmlns:android="http://schemas.android.com/apk/res/android">

  <item>
    <color android:color="@color/splash_background"/>
  </item>

  <item android:drawable="@drawable/icon"
        android:width="120dp"
        android:height="120dp"
        android:gravity="center" />

</layer-list>
```

And when I hover my mouse over the `@drawable/icon` value, Visual Studio shows my logo. So why then does the app show the Avalonia logo on the splash screen?
I've tried uninstalling the app and re-deploying, re-building, etc. to ensure there are no stale assets that could be used.

**Comment by maxkatz6** [comment]
We don't have any hardcoded splash screens, that icon should be the only one on the project.

**Reply by AArnott** [reply]
Thanks. I've built my app clean on CI and shipped multiple times, and the avalonia logo still appears on the splash screen. If anyone has bright ideas of where it's coming from, I'd love to hear it. Maybe it's hiding behind *mine* in a file that provides multiple resolutions?

**Reply by AArnott** [reply]
Could it be disguised as a vector graphic? What are these .xml files?
![image](https://github.com/AvaloniaUI/Avalonia/assets/3548/01a5b993-0405-4335-aefa-d0ebee5d160e)

**Comment by AArnott** [comment, marked as answer]
It turned out to be the `avalonia-anim.xml` vector graphic under `drawable-*` folders and referenced from `styles.xml`. Deleting those made my own logo appear on the splash screen.

**Comment by Awide** [comment]
I think this is still an issue, the Avalonia icon should not be a hardcoded vector in an .xml but it still is.
